web/ui/src/Pools/PoolsController.js

```diff
@@ -67,7 +67,8 @@
                     deferred.resolve();
                 })
                 .error(data => {
-                    $notification.create("Unable to load pools.", data.Detail).error();
+                    let message = data.Detail || data || "";
```

Comment:
if `data` is undefined here, trying to access `data.Detail` is going to throw an error. you may want to change this to match the other fix in this pr: ` let message = (data && data.Detail) || "";`